sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/transforms/SchemaTransformProvider.java

```diff
@@ -36,6 +36,21 @@ public interface SchemaTransformProvider {
   /** Returns an id that uniquely represents this transform. */
   String identifier();
 
+  // TODO(ahmedbu98): remove default when all existing SchemaTransformProviders implement this
+  // method
+  /**
+   * Returns a description which will be used to generate documentation for remote SDKs. This is
+   * reserved for how to build and use this SchemaTransform without getting into the specifics of
+   * each configuration field (that can go in the {@link
```

Comment:
"without getting into the specifics of each configuration field" might be too specific ? I think we should just skip these details and emphasize on the language-neutral part.